src/views/Register.vue

```diff
@@ -5,24 +5,24 @@
             <v-col cols="12"><h1>Komm in unser Netzwerk</h1></v-col>
         </v-row>
         <v-row>
-            <v-col cols="8">Gegenseitige Hilfe ist in der jetzigen Situation sehr wichtig. Gemeinsames Engagement sorgt zum einen zur wichtigen Entlastung medizinischer Kräfte wie aber auch zur besseren Versorgung von besorgten Bürgern und Patienten. </v-col>
+            <v-col cols="12" sm="10">Gegenseitige Hilfe ist in der jetzigen Situation sehr wichtig. Gemeinsames Engagement sorgt zum einen zur wichtigen Entlastung medizinischer Kräfte wie aber auch zur besseren Versorgung von besorgten Bürgern und Patienten. </v-col>
         </v-row>
         <v-row>
-            <v-col cols="4">
+            <v-col cols="12" sm="6" md="4">
```

Comment:
seh ich das richtig dass jede v-col 12 columns einnehmen würde ? vuetify arbeitet doch mit 12 sollte man also nicht anteilig 12 / anzahl v-cols (also je 4 cols) den platz reservieren? oder hab ich das konzept falsch verstanden?

Reply:
Siehst du (teilweise) richtig. In der mobilen Darstellung (Mobile first Konzept von vuetify) nimmt er pro Spalte die volle Breite ein (col="12"), wird man größer, wechselt es auf zweispaltig (sm="6") und dann auf dreispaltig (md="4").
Wie du richtig sagst, sind es 12 virtuelle Spalten und mit der Zahl kann ich definieren wie viel von den 12 Spalten er für eine col verwenden soll.